yowsup/demos/cli/layer.py

```diff
@@ -271,7 +271,7 @@ def disconnect(self):
 
     @clicmd("Quick login")
     def L(self):
-        return self.login(*self.getProp(YowAuthenticationProtocolLayer.PROP_CREDENTIALS))
+        return self.login(*self.getStack().getProp(YowAuthenticationProtocolLayer.PROP_CREDENTIALS))
```

Comment:
Sorry, i've never contributed in github before, I'm not sure how to pull. Anyway, the thing is command /L always make the cli crash because the property with the credentials is on the stack, not on the layer, check the /login code, its there.